java-api-core/src/com/kinvey/java/network/NetworkManager.java

```diff
@@ -362,6 +375,22 @@ public Get getBlocking(Query query, List<T> cachedItems, boolean deltaSetCaching
         return get;
     }
 
+
+    /**
+     * Get blocking with Delta Sync implementation
+     * @param query Query to get
+     * @param lastRequestTime Last request time
+     * @return
+     * @throws IOException
+     */
+    public QueryCacheGet queryCacheGetBlocking(Query query, String lastRequestTime) throws IOException {
+        Preconditions.checkNotNull(query);
```

Comment:
@yuliya-guseva in addition, you can also add the `@NonNull` (or `@Nullable`) annotation to the `query` parameter. This also helps `Kotlin` users to have a better interface to work with.

Reply:
Added